ui/js/services/jobfilters.js

```diff
@@ -407,15 +407,25 @@ treeherder.factory('thJobFilters', [
          */
         function getFieldFiltersArray() {
             const fieldFilters = [];
+            const clopt = thClassificationTypes.classificationOptions;
```

Comment:
Single source of truth, rather than duplicating a classification map in values.js.